components/ui/Button.tsx

```diff
@@ -107,3 +107,89 @@ interface ButtonProps {
   styles?: StyleProp<ViewStyle>;
   colors?: (string | number)[];
 }
+
+export const HalfButton: React.FC<ButtonProps> = (props) => {
```

Comment:
can we merge both the button component, or utilise the unchanged constants for both Button Components, because this creates code duplication.

Reply:
Good idea. I think we should, we can expose the width property or do it the way Twitter Bootstrap has done by dividing the entire viewport width into 12 column grid. I'll give it a shot.